static/app/views/issueDetails/sectionToggleButton.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {t} from 'sentry/locale';
+
+interface SectionToggleButtonProps {
+  isExpanded: boolean;
+  onExpandChange: (state: boolean) => void;
+}
+
+function SectionToggleButton({isExpanded, onExpandChange}: SectionToggleButtonProps) {
```

Comment:
Let's also forward props to the button, so folks can add useful things like `aria-controls`.